### PR TITLE
Fallback to Python's built-in json module if orjson unavailable

### PR DESCRIPTION
**Pull request by rbeeli**
`orjson` does not support all architectures. For those cases, if the `orjson` module import fails, the json serializer abstraction falls back to Python's built-in `json` module.

The two implementations have been adjusted/extended in order to output equal JSON strings, e.g. for date/datetime/numpy arrays etc. To ensure this, a new unit test `test_json.py` has been added.

### CODE REVIEW

File: nicegui/json/__init__.py [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
 in socketio.AsyncServer, which expects a module as parameter
 to override Python's default json module.
 """
 
-from nicegui.json.orjson_wrapper import dumps, loads
+try:
+    # orjson not available on all platforms, fallback to Python's json module if not available
+    import orjson
+    has_orjson = True
+except:
```
Comment (rodja) [inline nicegui/json/__init__.py:15]: maybe better `except ImportError`?

**Comment by falkoschindler** [comment]
This PR looks very good to me. Just two tiny things:

- There's another bare `except:` clause in test_json.py, which is discouraged by PEP8.
- The use of single and double quotes is pretty mixed up, sometimes on the same line. We try to use single quotes for regular strings (and `"""` for docstrings). But maybe that's just my OCD kicking in.

**Comment by psy0rz** [comment]
could you also a marker to exclude orjson if running with PyPy instead of regular cpython? 

I've tried installing rust so orjson would compile, but then it explicitly states:
```
        orjson does not support PyPy
```

**Comment by psy0rz** [comment]
i worked around it by using --no-deps and manually installing the other dependencies: after that it seems nicegui works under PyPy (did only do a few tests)

**Comment by evnchn** [comment]
@psy0rz I'm on it. check out #5163

**Comment by psy0rz** [comment]
Awesome thanks!!

**Comment by evnchn** [comment]
https://github.com/zauberzeug/nicegui/pull/5161

@psy0rz got it. But no tests on PyPy environment because something is wrong with the pipeline though (numpy cannot be imported for some reason).

**Pull request merged**